website/docs/configuration/package-curations.md

```diff
@@ -65,12 +66,12 @@ The structure of the curations file consist of one or more `id` entries:
     homepage_url: "http://example.com"
     binary_artifact:
       url: "http://example.com/binary.zip"
-      hash: 
+      hash:
```

Comment:
Ideally, do the trimming in a separate commit, as it's a non-functional change.

Reply:
reverted

Reply:
No need to revert, the change is good per se, but simply should go into a separate / preceding commit.

Reply:
done